config/initializers/devise.rb

```diff
@@ -264,12 +264,12 @@
 
   # Activate the test mode, useful when no real iChain is present, like in
   # testing and development environments
-  # config.ichain_test_mode = true
+  config.ichain_test_mode = true
 
```

Comment:
In this line you are uncommenting a line, that makes the setting enabled. The setting was not enabled before, which is why we are seeing this line being changed in your PR. 

If you did not intentionally make this change, please change the lines back to their original format.
Find me on IRC, if you have troubles with this.